src/main/java/com/todo/backend/BasicToDo.java

```diff
@@ -0,0 +1,48 @@
+package com.todo.backend;
+
+import java.time.Instant;
+
+import com.todo.backend.ToDo.Priority;
+
+/**
+ * Basic Properties for Create and Update ToDo items.
+ */
+public class BasicToDo {
+    // #region ################################ PROPERTIES
+    private String text;
+    private Priority priority;
+    private Instant due_date;
+    // #endregion
```

Comment:
unable to understand the word "region", in general I think each section explains themselves and there's no need of these comments